src/dashboard/install/README.md

```diff
@@ -1,4 +1,137 @@
-## Configuration files
+# Configuration
+
+- [Introduction](#introduction)
+- [Environment variables](#environment-variables)
+- [Other configuration files](#other-configuration-files)
+- [Logging configuration](#logging-configuration)
+
+## Introduction
+
+Dashboard has multiple sources of configuration. This is the full list ordered
+by precedence (with the last listed item winning prioritization):
+
+- Application defaults
+- Configuration file: `/etc/archivematica/archivematicaCommon/dbsettings`
+- Environment variables (always win precedence)
+
+## Environment variables
+
+The value of an environment variable is a string of characters. The
+configuration system coerces the value to the types supported:
+
+- `string` (e.g. `"foobar"`)
+- `int` (e.g. `"60"`)
+- `float` (e.g. `"1.20"`)
+- `boolean` where truth values can be represented as follows:
+    - True (enabled):  `"1"`, `"yes"`, `"true"` or `"on"`
+    - False (disabled): `"0"`, `"no"`, `"false"` or `"off"`
+
+Certain environment strings are mandatory, i.e. they don't have defaults and
+the application will refuse to start if the user does not give them one.
+
+Please be aware that Archivematica supports different types of distributions
+(Ubuntu/CentOS packages, Ansible or Docker images) and they may override some
+of these settings or provide values to mandatory fields.
+
+This is the full list of environment strings supported:
+
+- **`ARCHIVEMATICA_DASHBOARD_DASHBOARD_DJANGO_ALLOWED_HOSTS`**:
+    - **Description:** a list of strings representing the host/domain names that this Django site can serve. See [`ALLOWED_HOSTS`](https://docs.djangoproject.com/en/1.8/ref/settings/#allowed-hosts) for more details.
+    - **Type:** `string`
+    - :red_circle: **Mandatory!**
+
+- **`ARCHIVEMATICA_DASHBOARD_DASHBOARD_DJANGO_SECRET_KEY`**:
+    - **Description:** a secret key used for cryptographic signing. See [SECRET_KEY](https://docs.djangoproject.com/en/1.8/ref/settings/#secret-key) for more details.
+    - **Type:** `string`
+    - :red_circle: **Mandatory!**
+
+- **`ARCHIVEMATICA_DASHBOARD_DASHBOARD_SHARED_DIRECTORY`**:
+    - **Description:** location of the Archivematica Shared Directory.
+    - **Type:** `string`
+    - **Default:** `"/var/archivematica/sharedDirectory/"`
+
+- **`ARCHIVEMATICA_DASHBOARD_DASHBOARD_WATCH_DIRECTORY`**:
+    - **Description:** location of the Archivematica Watched Directory.
+    - **Type:** `string`
+    - **Default:** `"/var/archivematica/sharedDirectory/watchedDirectories/"`
+
```

Comment:
It's confusing that in the above bullet we see "watch directory", "watched directory" and "watched directories". Really, this is the path to the directory where all of the watched directories live. I don't know how exactly this can be improved. Maybe just changing the description to "location of the Archivematica Watched Directories." would be good.